src/App.css

```diff
@@ -1,24 +1,39 @@
+* {
```

Comment:
ide a reset részbe érdemes még felvenni `*::before` és `*::after` pszeudo elemeket is